Yoast/wordpress-seo
# Create subscription expiration notification

## Test instructions
For a test plugin to filter the subscriptions response, see https://github.com/Yoast/Wiki/wiki/Filter-the-subscriptions-response-in-the-addon-manager-request

Make sure the notifications ~~and banner~~ are shown as described in this comment, with the described intervals: https://github.com/Yoast/wordpress-seo-premium/issues/1650#issuecomment-378945453

~~Make sure the banner looks like this:
https://github.com/Yoast/wordpress-seo-premium/issues/1650#issuecomment-376572551~~

Make sure the notification is removed when no subscription is eligible for a notification.

This is the notification part of https://github.com/Yoast/wordpress-seo-premium/issues/1650

In order to test the Premium plugin, merge `wordpress-seo` in `wordpress-seo-premium`.
